_posts/2015-09-16-introduction-of-recommender-algorithms-for-e-commerce.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Introduction of Recommender Algorithms for E-Commerce"
+date: "2015-09-16"
+icon: Light-bulb
+categories: tech-stories
+authors: ["Pujit"]
+---
+
+##Introduction
+
+Recommender systems in e-commerce can be seen as a sales personal.  Like a good sales personal in a store helps increasing sales and building loyalty customer base, recommender system handles sales three ways: browser into buyers, cross-sell and loyalty.
+
+* **Browsers into buyers:** Visitors to a Web site often look over the site without ever purchasing anything. Recommender systems can help customers find products they wish to purchase.
+
+* **Cross-sell:** Recommender systems improve cross-sell by suggesting additional products for the customer to purchase. If the recommendations are good, the average order size should increase. For instance, a site might recommend additional products in the checkout process, based on those products already in the shopping cart.
+
+* **Loyalty:** In a world where a site’s competitors are only a click or two away, gaining customer loyalty is an essential business strategy. [4,5] Recommender systems improve loyalty by creating a value-added relationship between the site and the customer. Sites invest in learning about their users, use recommender systems to operationalize that learning, and present custom interfaces that match customer needs. Customers repay these sites by returning to the ones that best match their needs. [3]
+
+Hence, it is not surprising that Recommender Systems are essential part of e-commerce solutions in this era.  At present day, many recommender algorithms exist which can be employed into recommender systems built for e-commerce.  In this blog-post we will have high level overview of the classification of such recommender algorithms.
```

Comment:
@pshingala: Please mention only things which are possible with ePages at the moment. Otherwise our merchants will complain that we recommend things which they can't do with our software. For instance I am not aware that we have a real recommendation system which helps customers finding products. Same with cross-selling in the check out process or "custom interfaces".